### ISSUE TITLE
Show PV info doesn't work on Mac

### ISSUE THREAD

**Issue by vstadnytskyi** [pyepics/pyepics#196]
The show PV info function doesn't work on MacOS Mojave. It works fine on Windows 10.

```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/wx/core.py", line 3259, in <lambda>
    lambda event: event.callable(*event.args, **event.kw) )
  File "/Users/femto-13/All-Projects-on-femto/LaserLab/Software/pyepics/epics/wx/wxlib.py", line 162, in show_info
    dlg = wx.MessageDialog(self, self.pv.info,
  File "/Users/femto-13/All-Projects-on-femto/LaserLab/Software/pyepics/epics/pv.py", line 1062, in info
    return self._getinfo()
  File "/Users/femto-13/All-Projects-on-femto/LaserLab/Software/pyepics/epics/pv.py", line 895, in _getinfo
    out.append('      %s in file %s' % (cback.func_name,
AttributeError: 'function' object has no attribute 'func_name'
```
![UNADJUSTEDNONRAW_thumb_1c3a](https://user-images.githubusercontent.com/29989124/73587220-ebbfb900-447e-11ea-9312-6f3d5cc9afa1.jpg)

**Comment by newville** [comment]
@vstadnytskyi I believe this is fixed in master and version 3.4.1 with 4b204cdf01dd1e4552848ab52e15ee165ce478bf.   Can you verify?

**Comment by vstadnytskyi** [comment]
Yes. It is fixed.
